doc/users/cmake.html

```diff
@@ -470,6 +471,7 @@ <h3>Feature configuration</h3>
 DEAL_II_WITH_OPENCASCADE
 DEAL_II_WITH_P4EST
 DEAL_II_WITH_PETSC
+DEAL_II_PETSC_WITH_COMPLEX
```

Comment:
This is a feature that is detected while configuring. It should not be set by the user.

Reply:
That makes sense. I removed it :)